extensions.json

```
@@ -209,6 +209,13 @@
       "id": "csstools.postcss",
       "repository": "https://github.com/csstools/postcss-language"
     },
+    {
+      "id": "danielgjackson.auto-dark-mode-windows",
+      "repository": "https://github.com/danielgjackson/vscode-auto-dark-mode-windows",
+      "version": "1.0.8",
+      "checkout": "v1.0.8"
```

Unfortunately, there doesn't seem to be a `v1.0.8` tag for this repository: https://github.com/danielgjackson/vscode-auto-dark-mode-windows/tree/v1.0.8

Maybe there is no need to pin a specific version for this repository:

```suggestion
      "repository": "https://github.com/danielgjackson/vscode-auto-dark-mode-windows"
```